tls/s2n_key_update.c

```diff
@@ -146,3 +146,12 @@ int s2n_check_record_limit(struct s2n_connection *conn, struct s2n_blob *sequenc
 
     return S2N_SUCCESS;
 }
+
+int s2n_connection_request_key_update(struct s2n_connection *conn, bool peer_update_requested)
+{
+    POSIX_ENSURE_REF(conn);
+    /* s2n-tls does not currently support requesting key updates from peers */
+    POSIX_ENSURE(!peer_update_requested, S2N_ERR_T_USAGE);
```

Comment:
You can't set s2n_errno to a type. You need to set it to a specific error. I remember "S2N_ERR_UNIMPLEMENTED", but there's also just "S2N_ERR_INVALID_ARGUMENT"